docs/contributing/Pull_Requests.mdx

```diff
@@ -84,6 +103,15 @@ Once you've addressed all our feedback by making code changes and/or started a f
 
 Once the feedback is addressed, and the PR is approved, we'll ensure the branch is up to date with `main`, and merge it for you.
 
+#### Updating Over Time
+
+You generally don't need to keep merging commits from `main` into your PR branch.
+If you see merge conflicts or other intersections that worry you, then you can preemptively - but that's optional.
+
+If we think merge conflicts need to be resolved in order to merge and/or review a PR, we might do that for you as a courtesy _if_ we have time.
+If not, we may ask you to.
+If a PR spent an unusually long time to get a review, we'll try our best to find the time to help with merging.
```

Comment:
I still don't see how it's relevant to this section, which talks about "when/if I should merge main into my branch". Here is says "we will do it for you, or we will ask you to"—sure. Is the last sentence supposed to mean "if we have zero activity, trust that we will get there one day, and we'll get it to a mergeable state ourselves without bothering you again"? This still feels more relevant about review frequency in the previous section. (Also I remember we have a sentence about how long one should wait before pinging a maintainer, maybe it fits there)

Reply:
Gotcha, makes sense. Thanks! I just removed the sentence.